mycroft/skills/settings.py

```diff
@@ -328,6 +335,9 @@ def download(self, message=None):
 
         When used as a messagebus handler a message is passed but not used.
         """
+        if not self.sync_enabled:
+            LOG.debug("Skill settings sync is disabled, skipping download")
```

Comment:
This log message should be emitted once in the constructor as a INFO level message. Putting it here will unnecessarily pollute the logs when DEBUG is turned on